src/main/cms/cms_menu_vtx_smartaudio.c

```diff
@@ -144,6 +146,8 @@ if (saDevice.version == 2) {
         saCmsPitFMode = 1;
     else
         saCmsPitFMode = 0;
+} else if (saDevice.version == 3){
```

Comment:
There's a couple of places where missing space between `)` and `{` has snuck in again. Can you please fix this?
After that it will also need a rebase and squash into one commit.